From: Michael Olbrich <m.olbrich@pengutronix.de>
Date: Mon, 20 Oct 2025 18:06:48 +0200
Subject: [PATCH] HACK: don't use zlib

It's only used in libsolvext and we don't use that at all.

Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 5ba5ed51b14b..c63f85f6360c 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -144,7 +144,7 @@ IF (${have_system} STRGREATER xx)
     MESSAGE (FATAL_ERROR "Can only compile for one system type.")
 ENDIF (${have_system} STRGREATER xx)
 
-SET (ENABLE_ZLIB_COMPRESSION ON)
+SET (ENABLE_ZLIB_COMPRESSION OFF)
 IF (ENABLE_ARCHREPO OR ENABLE_DEBIAN)
 SET (ENABLE_LZMA_COMPRESSION ON)
 ENDIF (ENABLE_ARCHREPO OR ENABLE_DEBIAN)
